src/NServiceBus.Core/Features/FeatureStartupTaskController.cs

```diff
@@ -22,8 +22,19 @@ public Task Start(IBuilder builder, IMessageSession messageSession)
 
         public async Task Stop(IMessageSession messageSession)
         {
-            await instance.PerformStop(messageSession).ConfigureAwait(false);
-            DisposeIfNecessary(instance);
+            try
+            {
+                await instance.PerformStop(messageSession).ConfigureAwait(false);
+            }
+            catch
+            {
+                // TODO: Should we log here?
```

Comment:
>Should we log here? i

It seems reasonable to log exceptions during shutdown

>if yes DEBUG level?

Why only DEBUG, isn't this an indication of some issue => log a WARN?